rl4co/models/zoo/mdam/decoder.py

```diff
@@ -81,16 +94,14 @@ def __init__(
         ]
         self.project_out = nn.ModuleList(self.project_out)
 
-        self.dynamic_embedding = env_dynamic_embedding(
-            env_name, {"embedding_dim": embedding_dim}
-        )
+        self.dynamic_embedding = dynamic_embedding
 
         self.logit_attention = [
             LogitAttention(
                 embedding_dim,
                 num_heads,
                 mask_inner=mask_inner,
-                force_flash_attn=force_flash_attn,
+                # force_flash_attn=force_flash_attn,  # fix: not used
```

Comment:
We can delete this, too, it was left from the previous versions of FlashAttention which is now implemented natively in PyTorch

Reply:
delted